eventhubs/spring-cloud-azure-starter/spring-cloud-azure-sample-eventhubs-kafka/src/main/resources/application.yaml

```diff
@@ -3,20 +3,12 @@ spring:
     azure:
       eventhubs:
         connection-string: ${AZURE_EVENTHUBS_CONNECTION_STRING}
-##  Configure these commented out properties if you prefer using Azure Resource Manager
-#      profile:
-#        subscription-id: ${AZURE_SUBSCRIPTION_ID}
-#      eventhubs:
-#        namespace: ${AZURE_EVENTHUBS_NAMESPACE}
-#        resource:
-#          resource-group: ${AZURE_EVENTHUBS_RESOURCE_GROUP}
-##
     stream:
       function:
         definition: consume;supply
       bindings:
         consume-in-0:
-          destination: sample-eventhubs-kafka
+          destination: ${EVENTHUBS_KAFKA}
```

Comment:
why the spring cloud azure version in the source code is still 4.0.0-beta.2?

Reply:
that's another question, our sample is based on the released version, and  4.0.0-beta.3 has not been released yet.